config/config.md

```diff
@@ -242,6 +243,7 @@
 | `grpc.server_addr` | String | `127.0.0.1:4001` | The address advertised to the metasrv, and used for connections from outside the host.<br/>If left empty or unset, the server will automatically use the IP address of the first network interface<br/>on the host, with the same port number as the one specified in `grpc.bind_addr`. |
 | `grpc.runtime_size` | Integer | `8` | The number of server worker threads. |
 | `grpc.flight_compression` | String | `arrow_ipc` | Compression mode for frontend side Arrow IPC service. Available options:<br/>- `none`: disable all compression<br/>- `transport`: only enable gRPC transport compression (zstd)<br/>- `arrow_ipc`: only enable Arrow IPC compression (lz4)<br/>- `all`: enable all compression.<br/>Default to `none` |
+| `grpc.max_connection_age` | String | Unset | The maximum connection age for gRPC connection.<br/>The value can be a human-readable time string. For example: `10m` for ten minutes or `1h` for one hour.<br/>Refer to https://grpc.io/docs/guides/keepalive/ for more details. |
```

Comment:
Should we consider adding the `MAX_CONNECTION_AGE_GRACE` option for combined use?

Reply:
Unfortunately that's not supported by tonic, or rather, hyper, as of right now. If max connection age is reached, server simply sends a "GOAWAY" message to client, and wait to expect the client to re-establish the connection. If a mis-behavior client don't do that, server has currently no "timeout" method to use. See https://github.com/hyperium/tonic/issues/1820